### PR TITLE
Validation for Transfer Details & Confirm Completion components

### PR DESCRIPTION
**Pull request by dimak1**
*Issue #:* /bcgov/entity#13878
*Issue #:* /bcgov/entity#13960

*Description of changes:*
- Validation for Transfer Details and Confirm Completion components
- Unit tests

### Screenshots:

Screen Shot 2022-11-04 at 2 48 17 PM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/views/newMhrRegistration/MhrReviewConfirm.vue [hunk 1]
```diff
@@ -98,7 +98,6 @@ export default defineComponent({
     } = useMhrValidations(toRefs(getMhrRegistrationValidationModel.value))
 
     const localState = reactive({
-      isMhr: true,
```
Comment (cameron-eyds) [inline ppr-ui/src/views/newMhrRegistration/MhrReviewConfirm.vue:101]: 😆 Where did this property come from. 
Glad you removed!

**Review by cameron-eyds — APPROVED** [review]
Looks good!
Future state we might want to refactor the validations as the amount of Components that need validating grows. 

We could also modify our more global scroll to work everywhere. I do like the idea of Scrolling to the error border. Does it ever scroll past the section labels / headers etc?

**Review by ChasBoucher — APPROVED** [review]
Looks Good.

**Comment by dimak1** [comment]
> Looks good!
> 
> Future state we might want to refactor the validations as the amount of Components that need validating grows. 
> 
> 
> 
> We could also modify our more global scroll to work everywhere. I do like the idea of Scrolling to the error border. Does it ever scroll past the section labels / headers etc?

@cameron-eyds thanks, yea we might want to refactor and create global validations/scrolls/utils. I agree, we have so many implementations that we need to consolidate.

Scroll to error scrolls to first class, and stops at the top, it doesn't scrolls past it.

**Pull request merged**